app/views/api_keys/new.html.erb

```diff
@@ -0,0 +1,18 @@
+<% @title = t(".new_api_key") %>
+
+<div class="t-body">
+  <%= form_for @api_key, url: profile_api_keys_path do |f| %>
+    <%= label_tag :name, t("api_keys.index.name"), class: "form__label" %>
+    <%= f.text_field :name, class: "form__input", autocomplete: :off %>
+
+    <%= label_tag :scope, t("api_keys.index.scope"), class: "form__label" %>
+    <% ApiKey::API_SCOPES.each do |api_scope| %>
+      <div class = "form__checkbox__item">
+        <%= f.check_box "#{api_scope}", { class: "form__checkbox__input" } , "true", "false" %>
+        <%= label_tag api_scope, t("api_keys.index.#{api_scope}"), class: "form__checkbox__label" %>
```

Comment:
The text color for the label is too light/wrong color, can we match the color with the headers?